sdk/monitor/monitor-opentelemetry-exporter/src/platform/nodejs/httpSender.ts

```diff
@@ -21,7 +21,7 @@ export class HttpSender implements Sender {
     this._logger = this._exporterOptions.logger || new ConsoleLogger(LogLevel.ERROR);
     // Build endpoint using provided configuration or default values
     this._appInsightsClientOptions = {
-      endpoint: this._exporterOptions.endpointUrl + "/v" + this._exporterOptions.serviceApiVersion
+      host: this._exporterOptions.endpointUrl
```

Comment:
I'm guessing it's just implicit to the client now?

Reply:
yes, v2 would be added on the client